src/DotNetStatus/Common/AnalyzerInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Fx.Portability
+{
+    public class AnalyzerInfo
+    {
+        public string AnalyzerName { get; set; }
+        public string Id { get; set; }
+        public ICollection<CompatibilityRangeAttribute> Attributes { get; set; }
+        public bool IsCompatibilityDiagnostic { get; set; }
+
+        public int IdNumber
+        {
+            get
+            {
+                var matches = Regex.Match(Id, @"(\d+)");
+
+                return int.Parse(matches.Captures[0].Value);
+            }
+        }
+    }
+
+    public class CompatibilityRangeAttribute : Attribute
```

Comment:
I don't think this needs to derive from attribute. Can you check and remove if possible?